lib/ui/message-element.js

```diff
@@ -96,25 +92,35 @@ export class Message extends HTMLElement {
   }
   static getMultiLineMessage(message, includeLink) {
     const container = document.createElement('span')
-    const messageText = message.text || message.html
+    const messageEl = document.createElement('linter-multiline-message')
 
     container.className = 'linter-message-item'
+    messageEl.setAttribute('title', message.text)
 
-    if (includeLink && message.filePath) {
-      container.appendChild(Message.getLink(message))
-    }
+    message.text.split(NewLine).forEach(function(line, index) {
+      if (!line) return
 
-    const messageEl = document.createElement('linter-multiline-message')
-
-    for (let line of messageText.split(NewLine)) {
-      if (!line) continue
       const el = document.createElement('linter-message-line')
       el.textContent = line
       messageEl.appendChild(el)
-    }
+
+      // Render the link in the "title" line.
+      if (index === 0 && includeLink && message.filePath) {
+        messageEl.appendChild(Message.getLink(message))
+      }
+    })
 
     container.appendChild(messageEl)
 
+    messageEl.addEventListener('click', function(e) {
```

Comment:
Wait, did I write that text above? Disregard that, I feel like I am out of my mind these days. It's perfectly fine to make all the lines clickable. It **could** conflict with it when we'll make the text selectable in the future, but we'll solve it then instead of now.